recipe/meta.yaml

```
@@ -33,9 +33,9 @@ requirements:
     # This is here to work around what seems to be a bug with noarch
     # packages. In particular, flake8 requires this constraint on
     # importlib-metadata, but it's not enforced by conda nor mamba.
```

```suggestion
    # This is here to work around a bug with one of the flake8
    # builds.
```

See https://github.com/conda-forge/admin-requests/pull/375 for the fix. Unfortunately, it didn't work correctly so I have to inform the conda-forge team about it.